src/Microsoft.Dnx.Compilation.CSharp/RoslynProjectReference.cs

```diff
@@ -293,6 +271,26 @@ public DiagnosticResult EmitAssembly(string outputPath)
             }
         }
 
+        private EmitOptions GetEmitOptions()
+        {
+            var emitOptions = new EmitOptions();
+
+            var usePortablePdbString = Environment.GetEnvironmentVariable(EnvironmentNames.PortablePdb);
+
+            // Use portable pdbs if explicitly specified or the platform doesn't support pdb generation
+            var usePortablePdb = !_supportsPdbGeneration.Value ||
+                                 string.Equals(usePortablePdbString, "true", StringComparison.OrdinalIgnoreCase) ||
+                                 string.Equals(usePortablePdbString, "1", StringComparison.OrdinalIgnoreCase);
```

Comment:
Do we do this double-check thing anywhere else? Everywhere else I've seen only accepts "1" as a value. I'd rather just be clear on what the true value is than try to be unnecessarily flexible :)